master/getting-started/kubernetes/installation/hosted/rbac.yaml

```diff
@@ -63,6 +63,8 @@ rules:
   - apiGroups: ["projectcalico.org"]
     resources:
       - ippools
+      - globalbgpconfigs
+      - globalbgppeers
```

Comment:
bgp peers aren't created by calico/node.

do any of these need delete permissions?